### PR TITLE
Include version info in regkey check, and only have the regkey info once

### PR DESCRIPTION
**Pull request by champnic**
In response to this issue:
https://github.com/MicrosoftEdge/WebView2Feedback/issues/2331

We need to include info on the "0.0.0.0" version. I've also combined two sections that should be identical into a single section.

Preview: [https://review.docs.microsoft.com/en-us/microsoft-edge/webview2/concepts/distribution?branch=pr-en-us-1897](https://review.docs.microsoft.com/en-us/microsoft-edge/webview2/concepts/distribution?branch=pr-en-us-1897)

### CODE REVIEW

File: microsoft-edge/webview2/concepts/distribution.md [hunk 1]
```diff
@@ -212,36 +190,43 @@ If you have an offline deployment scenario, where app deployment has to work ent
...
-      ```text
+   If you run the following command from an elevated process or command prompt, it triggers a _per-machine_ install.  If you don't run the command from an elevated process or command prompt, a _per-user_ install will take place.  However, a _per-user_ install is automatically replaced by a _per-machine_ install, if a _per-machine_ Microsoft Edge Updater is in place.  A _per-machine_ Microsoft Edge Updater is provided as part of Microsoft Edge, except for the Canary preview channel of Microsoft Edge.  For more information, see [Installing the Runtime as per-machine or per-user](#installing-the-runtime-as-per-machine-or-per-user).<!-- since this link is provided, the present paragraph could be shortened -->
+
+   ```Shell
+   MicrosoftEdgeWebView2RuntimeInstaller{X64/X86/ARM64}.exe /silent /install
+   ```
+
+#### Detect if a suitable WebView2 Runtime is already installed
+
+To verify that a WebView2 Runtime is installed, use either of the following approaches:
```
Comment (vchapel) [inline microsoft-edge/webview2/concepts/distribution.md:207]: I might say ", use one of the following methods:" not a show stopper just my 2 cents.

**Review by vchapel — APPROVED** [review]
LGTM

File: microsoft-edge/webview2/concepts/distribution.md [hunk 2]
```diff
@@ -156,29 +156,7 @@ Use the following online deployment workflow or offline deployment workflow to e
...
-      The two registry locations to inspect on 32-bit Windows:
-
-      ```
-      HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\EdgeUpdate\Clients\{F3017226-FE2A-4295-8BDF-00C3A9A7E4C5}
-
-      HKEY_CURRENT_USER\Software\Microsoft\EdgeUpdate\Clients\{F3017226-FE2A-4295-8BDF-00C3A9A7E4C5}
-      ```
-
-   *  Approach 2: Run [GetAvailableCoreWebView2BrowserVersionString](/microsoft-edge/webview2/reference/win32/webview2-idl#getavailablecorewebview2browserversionstring) and evaluate whether the `versionInfo` is `nullptr`.  `nullptr` indicates that the WebView2 Runtime isn't installed.  This API returns version information for the WebView2 Runtime or for any installed preview channels of Microsoft Edge (Beta, Dev, or Canary).
+1. During your app setup, run a test to make sure that the WebView2 Runtime is already installed. See [Detect if a suitable WebView2 Runtime is already installed](#detect-if-a-suitable-webview2-runtime-is-already-installed)
```
Comment (mikehoffms) [inline microsoft-edge/webview2/concepts/distribution.md:159]:
```suggestion
1. During your app setup, run a test to make sure that the WebView2 Runtime is already installed. See [Detect if a suitable WebView2 Runtime is already installed](#detect-if-a-suitable-webview2-runtime-is-already-installed).
```

File: microsoft-edge/webview2/concepts/distribution.md [hunk 3]
```diff
@@ -212,36 +190,43 @@ If you have an offline deployment scenario, where app deployment has to work ent
 
 1. Include the Evergreen Standalone Installer in your app installer or updater.
 
-1. During your app setup, test whether the WebView2 Runtime is already installed, using either of the following approaches:
+1. During your app setup, run a test to make sure that the WebView2 Runtime is already installed. See [Detect if a suitable WebView2 Runtime is already installed](#detect-if-a-suitable-webview2-runtime-is-already-installed)
```
Comment (mikehoffms) [inline microsoft-edge/webview2/concepts/distribution.md:193]:
```suggestion
1. During your app setup, run a test to make sure that the WebView2 Runtime is already installed. See [Detect if a suitable WebView2 Runtime is already installed](#detect-if-a-suitable-webview2-runtime-is-already-installed).
```

File: microsoft-edge/webview2/concepts/distribution.md (hunk 2 above)
Comment (vchapel) [reply microsoft-edge/webview2/concepts/distribution.md]: of course!

**Review by mikehoffms — APPROVED** [review]

**Pull request merged**